projects/pgai/pgai/cli.py

```diff
@@ -247,6 +248,13 @@ def shutdown_handler(signum: int, _frame: Any):
     show_default=True,
     help="Exit immediately when an error occurs.",
 )
+@click.option(
+    "--doc-retries",
```

Comment:
Would it make sense to make this option agnostic of the loader? Like `--loading-retries` or `--load-retries` or similar?

Reply:
sounds good!